students/mattmaeda/session03/fruits.py

```diff
@@ -0,0 +1,14 @@
+def remove_fruit():
+    '''remove last or user specified fruit from list'''
+    fruits = ['Apples','Pears','Oranges','Peaches']
+    print(fruits)
+    del fruits[-1]
+    print(fruits)
+    choice = input("Which fruit should be removed? > ")
+    ## this works in ipython but not here...?
+    ## should remove all instances of choice
+    fruits2 = [x for x in fruits if x!= choice]
```

Comment:
nice list comprehension -- that should work fine.

note that the user will have needed to type the choice exactly the same way -- with no spaces or anything.